app/views/components/_share-links.html.erb

```diff
@@ -0,0 +1,18 @@
+<% title ||= 'Share this page' %>
+
+<div class="app-c-share-links-wrapper">
+  <h2><%= title %></h2>
+  <div class="app-c-share-links" data-module="track-share-button-clicks">
+    <%= link_to share_urls[:facebook],
```

Comment:
@fofr I kept this the same as the old share buttons which seem to generate the social media URLs from the current page and pass them into the partial/component via the content item?